exercises/bank-account/canonical-data.json

```diff
@@ -0,0 +1,372 @@
+{
+  "exercise": "bank-account",
+  "comments": [
+    "This exercise is a good candidate to practice concurrency, which",
+    "may not be available or possible in the implementing language track.",
+    "In that case, all tests can be executed on a single thread. If there",
+    "are tests that don't work using a single thread, these should be",
+    "marked with a concurrency scenario.",
+    "----------------------------------------------------------------",
+    "The overall expected value for the case should be the result of the last operation.",
+    "If any operation other than the last one also has an expected result,",
+    "that individual operation should have its expected value/error as a field.",
+    "For errors that are suppose to raise non fatal errors",
+    "Should have an extra field called `fatal` which holds a boolean"
```

Comment:
Is there any value in keeping this comment? I feel like this is more of a discussion than something a track maintainer should actually know about.